core/src/main/java/org/elasticsearch/index/query/MoreLikeThisQueryBuilder.java

```diff
@@ -364,7 +364,7 @@ public boolean equals(Object o) {
 
     // query formation parameters
     private String minimumShouldMatch = null;
-    private float boostTerms = -1;
+    private float boostTerms = 0;  // deactivated
```

Comment:
Can we use a constant here and use the same constant in the Parser to ensure that if the default is changed, the change is reflected consistently for the MLT query?